content/applications/hr/recruitment.rst

```diff
@@ -0,0 +1,730 @@
+appears, asking :guilabel:`Are you sure you want to remove this column?` Click :guilabel:`OK` to
+delete.
+
+.. image:: recruitment/delete-stage.png
+   :align: center
+   :alt: Delete a stage by accessing the drop-down menu from the gear icon.
+
+.. Important::
+   If there are applicants currently in the stage being deleted, an error pops up when attempting to
+   delete the stage, explaining that there are records in the stage which need to be deleted,
+   archived, or moved to a different stage before deleting.
+
+Email templates
+===============
+
+To communicate with the applicant, Odoo has several pre-configured email templates that can be used.
+The pre-configured email templates and when to use them are as follows:
+
+- :guilabel:`Applicant: Acknowledgement`: This template is used to let the applicant know that their
+  application was received. This email is automatically sent out once the applicant is in the
+  :guilabel:`First Interview` stage.
+- :guilabel:`Applicant: Interest`: This template is used to let the applicant know that they have
+  passed the :guilabel:`Initial Qualification` stage and they will be contacted to set up an
+  interview with the recruiter.
+- :guilabel:`Applicant: Not interested anymore`: This template is used when an applicant
+  communicates that they are no longer interested in the position, and thanks them for their time
+  and consideration.
+- :guilabel:`Applicant: Refuse`: This template is used when an applicant is no longer being
+  considered for the position.
+- :guilabel:`Applicant: Your Salary Package`: This template is used when offering an applicant a
+  position. This informs the applicant they can configure their salary package.
+
+.. note::
+   Email templates can be created, modified, and deleted to suit the needs of a business. For more
+   information on email templates, refer to the :doc:`../general/email_communication/email_template`
+   document.
+
+To send an email using an email template, click :guilabel:`Send message` in the chatter. A text box
+appears, as well as the applicant's email address. Click the :guilabel:`⤢ (Full composer)` icon in
+the bottom right corner of the :guilabel:`Send Message` tab in the chatter.
+
+.. image:: recruitment/full-composer.png
+   :align: center
+   :alt: Send an email using a template by clicking the full composer icon in the bottom right of
+         the send message tab in the chatter.
+
+An email pop-up loads, with the :guilabel:`Recipients` and :guilabel:`Subject` pre-populated. The
+applicant's email address is entered on the :guilabel:`Recipients` line, and the :guilabel:`Subject`
+is `Re: (Subject / Application Name)`. The email body is empty by default.
+
+To use an email template, click the drop-down next to the :guilabel:`Load template` field in the
+lower right section of the window. There are several pre-configured email templates to chose from.
```

Comment:
```suggestion
lower right section of the window. There are several pre-configured email templates to choose from.
```
Same with this one, it says resolved but is still showing up for me as "chose" not "choose".